README.md

```diff
@@ -1,23 +1,41 @@
 # Innopoints
 
-The application is based on the 
-[Sapper](https://sapper.svelte.dev/)
-template structure with some additions.
+This is the frontend for the Innopoints portal of Innopolis University.
 
-### Installation
+Built using:
+ - [Svelte](https://svelte.dev/)
+ - [Sapper](https://sapper.svelte.dev/)
+ - [Sass](https://sass-lang.com/)
+ - [Feather Icons](https://feathericons.com/)
+ - [Humaaans](https://humaaans.com/)
 
-To install dependencies and run the application, use:
-```
-npm install
-npm run dev
+## Running locally
+
+Ensure you have [Yarn](https://yarnpkg.com/) installed.
+To run the application locally, run the following:
+
+```bash
+yarn install
+yarn dev
 ```
 
-### Component system
+## Project structure
+
+There are three types of entities that build up this website:
+ - Pages ([`src/routes`](https://github.com/Innopoints/frontend/tree/master/src/routes))
+ - Containers ([`src/containers`](https://github.com/Innopoints/frontend/tree/master/src/containers))
+ - Components ([`src/components`](https://github.com/Innopoints/frontend/tree/master/src/components))
```

Comment:
Just FYI, you could use relative paths

Reply:
Huh? Last I checked, it wasn't possible. Do you have any example repos in mind?

Reply:
Our own https://github.com/Innopoints/innopoints/blob/master/README.md (the Nginx link)
And https://github.com/aabounegm/DMDI-Project/blob/master/README.md